os-virt.te

```diff
@@ -20,6 +21,14 @@ allow virtlogd_t systemd_logind_t:dbus send_msg;
 allow systemd_logind_t virtlogd_t:dbus send_msg;
 allow virtlogd_t systemd_logind_inhibit_var_run_t:fifo_file write;
 
+#============= virtlogd_t ==============
+# allow access to /var/lib/nova directories which are labled with container_file_t
+# This is required for https://issues.redhat.com//browse/OSPRH-960
+allow virtlogd_t container_file_t:dir { add_name getattr remove_name search write };
+allow virtlogd_t container_file_t:file { create open unlink };
```

Comment:
These two lines can be replaced with the following macro: `manage_files_pattern(virtlogd_t, container_file_t, container_file_t)`.